### ISSUE TITLE
[Feature] Values for adjusting nginx timeout

### ISSUE THREAD

**Issue by jcharcalla** [kubecost/cost-analyzer-helm-chart#2774]
### Problem Statement

I'm working with a customer that needs to adjust the nginx timeouts for `/model/` endpoints due to queries not completing on time. They already have significant CPU and Mem resources allocated and in testing increasing the timeout resolved there issue.

Currently these values are hardcoded [cost-analyzer-frontend-config-map-template.yaml](https://github.com/kubecost/cost-analyzer-helm-chart/blob/4ae1c4bd69057b8d808b5b2930fe9dd91ce23706/cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml) configMap making them difficult to manage.

The settings to be changed are: https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.106.4/cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml#L186-L188 and possibly others.

```
            proxy_connect_timeout       <value>;
            proxy_send_timeout          <value>;
            proxy_read_timeout          <value>;
```

### Solution Description

Modify the helm chart so these are set as values. Note: there is a bit of logic in the chart so the setting may occur in several places.

### Troubleshooting

- [X] I have read and followed the [issue guidelines](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/ISSUE_GUIDELINES.md) and this is a feature request only for the Helm chart.
- [X] I have searched other issues in this repository and mine is not recorded.

**Comment by jcharcalla** [comment]
They have asked if the fix can be back ported to 1.106.x as they are not ready to upgrade at this time.

**Comment by jessegoodier** [comment]
FYI: this is done. Just waiting on next release.
